src/lib/tabs/tab-group.ts

```diff
@@ -269,5 +272,6 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
   _removeTabBodyWrapperHeight(): void {
     this._tabBodyWrapperHeight = this._tabBodyWrapper.nativeElement.clientHeight;
     this._renderer.setStyle(this._tabBodyWrapper.nativeElement, 'height', '');
+    this.animationDone.emit();
```

Comment:
Unit test? Even with `NoopAnimationModule`, it should still fire the event

Reply:
It looks like the `_onTranslateTabComplete` function is getting called in `tab-body.ts`, however, the `onCentered` event that is captured in `tab-group.ts` in the `_removeTabBodyWrapperHeight` method is not getting triggered in the tests. Any thoughts?

Reply:
What kind of flushing / waiting does the test use?

Reply:
Here is the test I've written for this:

```
fit('should fire animation done event', async (() => {
  fixture.detectChanges();

  spyOn(fixture.componentInstance, 'animationDone');
  let tabLabel = fixture.debugElement.queryAll(By.css('.mat-tab-label'))[1];
  tabLabel.nativeElement.click();
  fixture.detectChanges();

  fixture.whenStable().then(() => {
    fixture.detectChanges();
    expect(fixture.componentInstance.animationDone).toHaveBeenCalled();
  });
}));
```